guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<blockquote>remove this line <em><u>Also the sequence length  here is 12 since being monthly data, the seasonal pattern repeats after 12 months.</u></em></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>